plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/edit/PostgreRoleManager.java

```diff
@@ -148,4 +152,21 @@ private void addRoleOptions(StringBuilder script, PostgreRole role, NestedObject
         }
         script.append(options);
     }
+
+    @Override
+    protected void addObjectExtraActions(
+        DBRProgressMonitor monitor,
+        DBCExecutionContext executionContext,
+        List<DBEPersistAction> actions,
+        NestedObjectCommand<PostgreRole, PropertyHandler> command,
+        Map<String, Object> options)
+    {
```

Comment:
```suggestion
        Map<String, Object> options
    ) {
```

Reply:
And now we here again.

Reply:
Make CheckStyle happy.
Make me happy.

Reply:
Nope. Just nope. We discussed it many times.

Map<String, Object> options) <-- end of the arguments list
{ <-- start of the method body

A matter of taste.

Reply:
I thought we agreed to use the proposed pattern last time :thinking_face:
It's sad